turbo/jsonrpc/eth_api.go

```diff
@@ -383,6 +386,7 @@ type APIImpl struct {
 	LogsMaxRange                  uint64
 	DisableStateRootCheck         bool
 	DisableVirtualCounters        bool
+	UsingEthereumHardfork         bool
```

Comment:
just for completeness, we might not need this if we can write the property into the DB alongside the rest of the chain config.